client/my-sites/sidebar/use-site-menu-items.js

```diff
@@ -169,7 +164,7 @@ const useSiteMenuItems = () => {
 		shouldShowMailboxes,
 		shouldShowAddOns,
 		showSiteMonitoring: isAtomic,
-		showGithubDeployments: gitHubDeploymentsAvailable?.available ?? false,
+		showGithubDeployments: isAtomic,
```

Comment:
We should always show it since we're offering the user to transfer the site to Atomic.

Reply:
Updated, I've removed the condition.